[RUM-3349] RUM Browser consent management doc (#22087)
DataDog/documentation: content/en/real_user_monitoring/browser/advanced_configuration.md

For a sampled out session, all page views and associated telemetry for that session are not collected.

## User tracking consent

> To be compliant with GDPR, CCPA and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.

Reviewer: ```suggestion
To be compliant with GDPR, CCPA, and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.
```

> To be compliant with the GDPR, CCPA and similar regulations, the RUM Browser SDK allows to provide the tracking consent value at initialization.

Reviewer: ```suggestion
To be compliant with the GDPR, CCPA and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.
```

The `trackingConsent` initialization parameter can be one of the following values:

1. `"granted"`: The RUM Browser SDK starts collecting data and sends it to Datadog.
2. `"not-granted"`: The RUM Browser SDK does not collect any data.

To change the tracking consent value after the RUM Browser SDK is initialized, use the `setTrackingConsent()` API call. The RUM Browser SDK changes its behavior according to the new value:

> * when changed from `"granted"` to `"not-granted"`, the RUM session is stopped, and no data will be sent anymore.

Reviewer: ```suggestion
* when changed from `"granted"` to `"not-granted"`, the RUM session is stopped, data is no longer sent to Datadog.
```

> * when changed from `"not-granted"` to `"granted"`, a new RUM session is be created if no previous session is active, and data collection is be started again.

Reviewer: ```suggestion
* when changed from `"not-granted"` to `"granted"`, a new RUM session is created if no previous session is active, and data collection resumes.
```
